packages/react-dev-utils/package.json

```diff
@@ -71,7 +72,7 @@
     "loader-utils": "1.2.3",
     "open": "^7.0.2",
     "pkg-up": "3.1.0",
-    "react-error-overlay": "^6.0.7",
+    "react-error-overlay": "^6.0.8",
```

Comment:
Again this should change...

Reply:
Might be since I rebased the latest release

Reply:
This will have been lerna when I did the publish